tools/packaging/common/envoy_bootstrap.json

```diff
@@ -145,9 +145,44 @@
       "transport_api_version": "V3",
       "grpc_services": [
         {
+        {{ if .call_credentials }}
```

Comment:
Do we want to do this now or just get the XDS proxy in? We can merge that right now anyways

Reply:
I think both are valuable. XDS proxy will likely be experimental at first - this is stable code ( tested with TD )